src/appmixer/monday/package.json

```diff
@@ -1,8 +1,9 @@
 {
     "name": "appmixer.monday",
     "author": "Harsha Surisetty <harsha@client.io>",
-    "version": "0.1.0",
+    "version": "2.0.0",
```

Comment:
Version should be consistent with the bundle.json version I suppose? Not sure which of the two fits into your versioning, but we moved from 1.x to 2. due to the breaking API change

Reply:
Appmixer engine reads the version from the `bundle.json` and ignores the `package.json`. However, it could make sense outside of the Appmixer.